js/game.js

```diff
@@ -81,7 +90,7 @@ startgame = function () {
 	init();
 	function setSpeed(snakeSpeed) {
 		currentSpeed = setInterval(() => {
-			console.log("snake speed in set interval " + snakeSpeed);
+			// console.log("snake speed in set interval " + snakeSpeed);
```

Comment:
remove all console.log and unwanted commented code.

Reply:
done.